client/state/action-types.js

```diff
@@ -564,6 +564,8 @@ export const PREVIEW_TYPE_RESET = 'PREVIEW_TYPE_RESET';
 export const PREVIEW_TYPE_SET = 'PREVIEW_TYPE_SET';
 export const PREVIEW_URL_CLEAR = 'PREVIEW_URL_CLEAR';
 export const PREVIEW_URL_SET = 'PREVIEW_URL_SET';
+export const PRIVACY_POLICY_RECEIVE = 'PRIVACY_POLICY_RECEIVE';
```

Comment:
I would recommend `PRIVACY_POLICY_ADD`. If we trim down the data in the dispatch to what we want, eg: data.entities.automattic.